src/enums.h

```diff
@@ -245,5 +245,15 @@ typedef enum {
 } htcp_clr_reason;
 #endif /* USE_HTCP */
 
+typedef enum {
+#if USE_OPENSSL
+    xaStepTlsBump1,
```

Comment:
Let's keep the _whole_ common prefix low-cased and simple:

```suggestion
    xstepTlsBump1,
```

Reply:
commit 6f3b3f0bba